en/style/typography.md

```diff
@@ -30,42 +34,64 @@ These styles also determine the type of text element and typography class that w
   - `igx-typography__caption` for the caption typography style.
   - `igx-typography__overline` for the overline typography style.
 
-These styles can be modified further via the Sketch plugin to change the look of those elements via theming on the application level.
+These styles can be modified further via the **Indigo.Design** plugin to change the look of those elements via theming on the application level.
```

Comment:
**Indigo.Design** plugins for Figma and Sketch maybe sounds better

Reply:
Definitely, but there is a plugin for Adobe XD as well and this is why I left it like this